src/tensor_ops.jl

```diff
@@ -152,17 +157,22 @@ Computes the trace of a second or fourth order tensor.
     @code :(return s)
 end
 
+const vol = trace
+
+
 ########
 # Mean #
 ########
+
 Base.mean{dim}(S::SecondOrderTensor{dim}) = trace(S) / dim
```

Comment:
Yes, the dim is still needed right?

`Base.mean{dim}(S::SecondOrderTensor{dim}) = trace(S) / 3`

Since `SecondOrderTensor` will be abstract, have I understood it correctly?